lib/lob/resources/resource_base.rb

```diff
@@ -100,7 +100,7 @@ def submit(request)
       def handle_api_error(error)
         begin
           response = JSON.parse(error.http_body.to_s)
-          message = response.fetch("error").fetch("message")
+          message = response.dig("error", "message")
```

Comment:
`KeyError` will probably never get triggered, since the new statement is the equivalent of `response.fetch("error", {}).fetch("message")`

Reply:
Updated back to `fetch` functionality for strictness-sake: 1b1bb83c